src/lib/viewers/box3d/image360/__tests__/Image360Controls-test.tsx

```diff
@@ -24,11 +22,10 @@ describe('lib/viewers/box3d/image360/Image360Controls', () => {
                 onVrToggle,
             });
 
-            expect(wrapper.find(VrToggleControl).props()).toMatchObject({
-                isVrShown: false,
-                onVrToggle,
-            });
-            expect(wrapper.find(FullscreenToggle).prop('onFullscreenToggle')).toEqual(onFullscreenToggle);
+            await expect(wrapper.queryByTitle('Toggle VR display')).toBe(null);
+
+            await wrapper.queryByTitle('Enter fullscreen')?.click();
```

Comment:
What is `click()`?
Recommended way of working with events is to use `fireEvent` (a few use cases) and `userEvent` (majority of the time) [@testing-library/user-event](https://testing-library.com/docs/user-event/intro).

Source: [Firing Events at Testing Library docs](https://testing-library.com/docs/dom-testing-library/api-events/)